contrib/lib/src/serve.rs

```diff
@@ -14,12 +14,12 @@
 //! features = ["serve"]
 //! ```
 
-use std::path::{PathBuf, Path};
+use std::path::{Path, PathBuf};
```

Comment:
`rustfmt` strikes again... would you mind reverting the formatting-only changes so it's easier to review what did change?

Reply:
Do you have any special configuration that I'm missing? If I run `cargo fmt` in the root, `git status` shows me more than 200 modified files.

Reply:
You shouldn't be running it at all, is my point. We can't configure `cargo fmt` in a way that is compatible with the current style of the codebase and also consistent, as much as we want it to work.